keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/telemetry/TelemetryEvent.kt

```diff
@@ -2,23 +2,16 @@ package com.netflix.spinnaker.keel.telemetry
 
 import com.netflix.spinnaker.keel.api.ApiVersion
 import com.netflix.spinnaker.keel.api.ArtifactType
-import com.netflix.spinnaker.keel.api.Resource
 import com.netflix.spinnaker.keel.api.ResourceId
-import com.netflix.spinnaker.keel.api.id
 
 sealed class TelemetryEvent
 
 data class ResourceCheckSkipped(
   val apiVersion: ApiVersion,
   val kind: String,
-  val id: ResourceId
-) : TelemetryEvent() {
-  constructor(resource: Resource<*>) : this(
-    resource.apiVersion,
-    resource.kind,
-    resource.id
-  )
-}
+  val id: ResourceId,
+  val skipper: String = "unknown"
```

Comment:
What's this name all about?

Reply:
Who skipped the check. I added this so that we can use this metric to track the unhappy resources because they'll all have been skipped by the same thing.